source/mobile/index.html.md

```diff
@@ -0,0 +1,32 @@
+---
+title: GOV.UK mobile app documentation for developers
+weight: 1
+layout: multipage_layout
+---
+
+<div class="govuk-notification-banner govuk-!-margin-top-6 govuk-!-margin-bottom-6" role="region" aria-labelledby="govuk-notification-banner-title" data-module="govuk-notification-banner">
+  <div class="govuk-notification-banner__header">
+    <h2 class="govuk-notification-banner__title" id="govuk-notification-banner-title">
+      Important
+    </h2>
+  </div>
+  <div class="govuk-notification-banner__content">
+    <p class="govuk-notification-banner__heading">
+      The mobile app is a work in progress.
+    </p>
+  </div>
+</div>
+
+# GOV.UK mobile app documentation for developers
+
+Use this documentation to find information about the GOV.UK mobile app:
+
+- general information relating to general mobile application topics that are not specific to either the Android or iOS platforms
+- information relating to iOS specific topics
+- information relating to Android specific topics
+
+## Introduction
+
+This site contains documentation relating to the GOV.UK mobile application.
+
+It exists to aid developers on the GOV.UK mobile team, but is available to anyone who may find our approach useful.
```

Comment:
Maybe we don't need this section right now? As the first sentence is a dupe of the information above and not sure the second adds much.

Reply:
Content updated - resolving